sympy/geometry/parabola.py

```diff
@@ -376,6 +376,19 @@ def p_parameter(self):
             p = sign(self.focus.args[1] + y)
         return p * self.focal_length
 
+    @property
+    def latus_rectum(self):
+        """latus rectum of parabola- a special chord that passes through focus
+           Returns
+           =======
+           equation of latus rectum
+           >>> p1 = Parabola(Point(0, 0), Line(Point(5, 8), Point(7, 8))
```

Comment:
```suggestion

           Examples
           =======

           >>> p1 = Parabola(Point(0, 0), Line(Point(5, 8), Point(7, 8)))
```

Reply:
If you use @smichr 's suggestion, I think it'd be better to change the heading to examples and leave a space above and below the heading.

Also, there should be anther closing parenthesis in the definition of p1.